src/interfaces/Ownership.sol

```diff
@@ -1,7 +0,0 @@
-//SPDX-License-Identifier: MIT
```

Comment:
How does the diamond handle a transfer of ownership? Not sure I am aware and this was previously used to change the ownership of the deployed proxies.

Reply:
This diamond implementation (mudgen's version) has a separate facet called `OwnershipFacet` for ownership management and it is currently present in ours as well, we're using it for all admin functions in the AdminFacet.
We can just call `IERC173(address(proxy)).transferOwnership(newOwner)` for change of ownership.